src/components/gridView/index.tsx

```diff
@@ -166,12 +155,7 @@ class GridView extends UIComponent<GridViewProps, GridViewState> {
 
     const imageBorderRadius = _.flow(_.first, item => _.get(item, 'imageProps.borderRadius'))(items);
     return (
-      <View
-        style={[
-          styles.overlayContainer,
-          {backgroundColor: Colors.rgba(overlayColor, 0.6), borderRadius: imageBorderRadius}
-        ]}
-      >
+      <View style={[styles.overlayContainer, {backgroundColor: lastItemOverlayColor, borderRadius: imageBorderRadius}]}>
```

Comment:
Here you are changing the ui (it's not breaking change, but a UI breaking change), since the user now needs to pass the color with the alpha, while we did it internally before.

Reply:
I agree. 
I'm fine with breaking them since it's pretty small, moreover it's a pretty new component and it appear we had issues there anyway. 
We had old code that depended on place color logic, which obviously not relevant to the component code. 
And generally I think it's wrong to decide the alpha for them, therefor they should pass the color as they wish. 

I don't think this small break worth having a migration for it..